nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```
@@ -147,11 +147,8 @@ in {
         default = false;
         type = types.bool;
         description = lib.mdDoc ''
-          Make MemTest86 available from the systemd-boot menu. MemTest86 is a
-          program for testing memory.  MemTest86 is an unfree program, so
-          this requires `allowUnfree` to be set to
-          `true`.
-        '';
+          Make MemTest86+ available from the systemd-boot menu. MemTest86+ is a
+          program for testing memory.'';
```

Trivial nit, if you can't handle it (time constraint, ate too much, any reason) do tell and I'll handle that.

```suggestion
          Make MemTest86+ available from the systemd-boot menu. MemTest86+ is a
          program for testing memory.
        '';
```